### DISCUSSION TITLE
Script for automatic ordering and provisioning of hetzner-servers

### DISCUSSION THREAD

**Discussion by Subito** [depenguin-me/depenguin-run#88] [Show and tell]
Hi!

we just open sourced https://github.com/netzkommune/depenguin-provision which rents servers from hetzner and automatically provisions them with depenguin. We used it a few dozen times in the last weeks, and it was a great time-saver. 

Especially with servers we just needed for a short time-period, it was great to just fire off the script, and after a few minutes login to a ready-made FreeBSD installation. 

We are also starting to use this on other providers and had mostly success so far. It requires adding a manual installerconfig with network-configuration, but apart from that its still a lot faster then executing all the steps in order by hand.

Hope you find this useful,

Cheers,
Filias

**Comment by grembo** [comment]
Sounds cool, hope to find the time to give it a shot

**Comment by bretton** [comment]
that's awesome, make sure to update
```
image_url=https://depenguin.me/files/mfsbsd-14.0-RELEASE-rc5-amd64.iso
```
to 
```
image_url=https://depenguin.me/files/mfsbsd-14.0-RELEASE-amd64.iso
```

**Reply by Subito** [reply]
good catch! Thanks!
